README.md

```diff
@@ -12,24 +12,15 @@ For tutorials, samples, quick starts, and other documentation, go to [Azure for
 
 ## Packages available
 
-### Client: November 2019 Releases
+For the most up-to-date information about 
+
+### Client
+
 New wave of packages that we are announcing as **GA** and several that are currently releasing in **preview**. These libraries follow the [Azure SDK Design Guidelines for .NET](https://azure.github.io/azure-sdk/dotnet/guidelines/) and share a number of core features such as HTTP retries, logging, transport protocols, authentication protocols, etc., so that once you learn how to use these features in one client library, you will know how to use them in other client libraries. You can learn about these shared features at [Azure.Core](sdk/core/Azure.Core/README.md).
 
 These preview libraries can be easily identified by their folder, package, and namespaces names starting with 'Azure', e.g. Azure.Storage.Blobs.
```

Comment:
This pattern would be applicable to all of the track two libraries,  maybe we want to update the verbiage to be inclusive of the GA and preview.  We may also want to avoid using terms like `easily`, as not everyone may find it easy and we don't want to belittle them.

Perhaps consider something like:  
    ```These new client libraries can be identified by the naming used for their folder, package, and namespace.  Each will start with `Azure`, followed by the service category, and then the name of the service.  For example [`Azure.Storage.Blobs`](./sdk/core/Azure.Storage.Blobs).```